e2e/ansible/playbooks/resiliency/test-kubelet-service-stop/test-kubelet-service-stop.yml

```diff
@@ -0,0 +1,227 @@
+         include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/regex_task.yml"
+         vars:
+           path: "{{ result_kube_home.stdout }}/percona.yaml"
+           regex1: "{{replace_item}}"
+           regex2: "{{replace_with}}"
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+
+       - name: Create namespace to deploy application
+         shell: source ~/.profile; kubectl create ns {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{ groups['kubernetes-kubemasters'].0 }}"
+
+       - name: Create a configmap with the liveness sql script
+         shell: source ~/.profile; kubectl create configmap sqltest --from-file={{result_kube_home.stdout}}/{{ percona_files.1 }} -n {{ namespace }}
+         args:
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         register: result
+         failed_when: "'configmap' and 'created' not in result.stdout"
+
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_task.yml"
+         vars:
+           app_yml: "{{ percona_files.0 }}"
+           ns: "{{ namespace }}"
+
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+            ns: "{{ namespace }}"
+            app: percona
+
+       - name: Wait for 120s to ensure liveness check starts
+         wait_for:
+           timeout: 120
+
+       - name: Get the number of nodes in the cluster
+         shell: kubectl get nodes | grep '<none>' | wc -l
+         args:
+           executable: /bin/bash
+         register: node_out
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Fetch the node count from stdout
+         set_fact:
+            node_count: " {{ node_out.stdout}}"
+
+       - name: Get the number of nodes that are in ready state
+         shell: source ~/.profile; kubectl get nodes | grep 'Ready' | grep 'none' | wc -l
+         args:
+           executable: /bin/bash
+         register: ready_node_cout
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+       - debug:
+           msg: "All the nodes are in ready state"
+         when: (node_count)| int == (ready_node_cout.stdout)| int
+
+       - name: Stop kubelet service in one of the node
+         shell: source ~/.profile; systemctl stop kubelet.service
+         args:
+           executable: /bin/bash
+         register: result
+         become: True
+         delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
+         changed_when: True
+
+
+       - name: check the node is in NotReady state after kubelet service is stopped
+         shell: source ~/.profile; kubectl get nodes | grep NotReady | wc -l
+         args:
+           executable: /bin/bash
+         register: result
+         until: "'1' in result.stdout"
+         delay: 60
+         retries: 5
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+         changed_when: True
+
```

Comment:
@Swarnalatha-K Can we check the application status when the kubelet service is stopped?